portal/patient_self_checkin/mySQL_Connector.php

```diff
@@ -0,0 +1,13 @@
+<?php
+$ignoreAuth = true;
+require_once "../../interface/globals.php";
+require_once $GLOBALS['webserver_root'] . '/sites/default/sqlconf.php';
```

Comment:
Check out my comment in portal/patient_self_checkin/step5.php where I converted block of code to use OpenEMR's sqlStatement() function.
Main function used (which are from the library Robert linked above) are:
sqlStatement() (main function that is used)
sqlQuery() (this function only returns 1 row in array form)
sqlInsert() (if need to get an id after doing an INSERT, then this is function to use)